### PR TITLE
I've added a html page with information.

### PR DESCRIPTION
**Pull request by Ldavis603**

### CODE REVIEW

**Review by janet-phan — APPROVED** [review]
looks good!

File: group2.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <title>Group 2</title>
+    <h1>Nicole</h1>
```
Comment (Npcodes1) [inline group2.html:25]: Nicole likes long walks to the kitchen and eating all things chocolate.

**Review by janet-phan — COMMENTED** [review]
Can you just add in my paragraph "Janet currently lives in Atlanta. She loves boba, korean bbq, and working on her podcast with her sister!"

File: group2.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <h1>Janet</h1>
+    <h1>Anaidys</h1>
```
Comment (janaidys) [inline group2.html:29]: Hey y'all. I'm from Boston, MA. I'm currently a high school Geometry teacher at KIPP Academy Lynn Collegiate. After 5 years of teaching,  I'm looking to make a career switch into tech.

**Comment by janet-phan** [comment]
Can you overwrite your header with the new HTML? 
```
`<nav>
                <ul>
                    <li>
                        <a href="#">Home</a>
                    </li>
                    <li>
                        <a href="group2.html">Team 2</a>
                    </li>
                    <li>
                        <a href="#">Contact</a>
                    </li>
                </ul>
            </nav>
```

**Review by Npcodes1 — APPROVED** [review]
Looks good!

**Pull request merged**